src/routes/api/ghes-app-verification/ghes-app-verify-get-apps.ts

```diff
@@ -36,7 +37,7 @@ export const GHESVerifyGetApps = async (req: Request, res: Response): Promise<vo
 			const output = await runCurl({
 				fullUrl: `${gitHubAppClient.restApiUrl}/app`,
 				method: "GET",
-				authorization: `Bearer ${AppTokenHolder.createAppJwt(await app.getDecryptedPrivateKey(installation.jiraHost), String(app.appId)).token}`
+				authorization: `Bearer ${AppTokenHolder.createAppJwt(await app.getDecryptedPrivateKey(installation.jiraHost), String(app.appId), FIVE_MINUTES_IN_MILLI_SEC).token}`
```

Comment:
Why are we not feature flagging this too?

Reply:
I realize that this is for the API endpoints that we use, but we can't test the exp time set in FF from this endpoint , can we? 
So if the customer is still facing this issue, we can't test the value in the FF through this endpoint cause its set to 5 minutes.